vars/fcosKolaTestIso.groovy

```diff
@@ -34,6 +34,7 @@ def call(params = [:]) {
             } else {
                 shwrap("cd ${cosaDir}  && kola testiso -S ${extraArgs} --output-dir tmp/kola-testiso-metal")
             }
+            shwrap("cd ${cosaDir}  && kola testiso -S --add-nm-keyfile --scenarios iso-install --output-dir tmp/kola-testiso-metal")
```

Comment:
Ughh, still debating whether it should've just been a separate scenario [when I implemented this](https://github.com/coreos/coreos-assembler/pull/2614). It's orthogonal, but the way the harness handles scenarios makes it easier to use.

Hmm, let's try the suggestion above and see how it goes. Open to switch to scenario down the line if it's easier.

Reply:
> 

Yes it will overwrite the previous results. I'll make the changes to output to a different dir.
So far it looks okay; it's definitely easier to test a scenario than testing one particular switch. Also, don't think we need to convert it to a scenario (at least for now though) :smiley:

Reply:
if we did convert it to a scenario we wouldn't need to change this file at all?

Reply:
> if we did convert it to a scenario we wouldn't need to change this file at all?

Assuming we add this new scenario to the the default set, indeed. But we'd have to decide which ISO-based scenarios to duplicate. Likely `iso-install` makes the most sense. But it'd be nice to still be able to enable it when testing the others.

Hmm, maybe we can keep the switch but just add a `iso-install-nm` scenario which forces the switch on?